src/Workspaces/CoreTest/FindReferencesTests.cs

```diff
@@ -352,9 +369,9 @@ public interface I
     {
         System.Uri Get();
     }
-}", NetStandard20Ref);
+}", NetStandard20.References.All);
 
-            solution = AddProjectWithMetadataReferences(solution, "DesktopProject", LanguageNames.CSharp, @"
+            solution = AddProjectWithMetadataReferences(solution, "NetCoreProject", LanguageNames.CSharp, @"
```

Comment:
This changes the test here from being a `net461` project referencing a `netstandard2.0` to `net8.0` referencing `netstandard2.0`. That actually matches the [original issue](https://github.com/dotnet/roslyn/issues/28827) closer by my reading. It is still under the hood though verifying that we can map symbols between the two projects when the underlying references substantially change and we have to deal with mapped symbols. 

Waiting for @sharwell, @jasonmalinowski or @CyrusNajmabadi to sign off on this or let me know if I missed a subtle issue about this test.

Reply:
We probably went with testing netstandard and .NET Framework since they'd be most "different" in terms of types being in different assemblies. Is there a reason we can't keep this a .NET Framework ref?

Reply:
> Is there a reason we can't keep this a .NET Framework ref?

It's difficult to keep the `net461` reference because of how the mapping between `netstandard2.0` to `net461` works under the hood. It effectively downloads / loads a deprecated NuPkg that adds a couple dozen facade assemblies to the build. This is needed because `netstandard2.0` was somewhat ret-conned into `net461`. If there were a lot of tests that did the `netstandard -> net461` mapping I'd be inclined to get that all mapped out with basic ref assemblies and use it. But this the _only_ test that does this. 

> We probably went with testing netstandard and .NET Framework since they'd be most "different" in terms of types being in different assemblies

I assure you they are quite different in `netstandard2.0` and any modern .NET core TFM. For example any core primitive type goes from being defined in `netstandard.dll` to `System.Runtime.dll`. They are, quite unfortunately, substantially different.

Reply:
> or example any core primitive type goes from being defined in netstandard.dll to System.Runtime.dll

OK that would worry me less here; as long as we have some test that is asserting that we can handle the cases where System.Object comes from different assemblies, we're probably good.